pkg/env/env.go

```diff
@@ -62,30 +62,32 @@ func (e *Env) Environ() (vars []string) {
 	return vars
 }
 
-func (e *Env) getAndSplitPath() []string {
+// GetPathParts gets paths in PATH environment variable
+func (e *Env) GetPathParts() []string {
```

Comment:
Nit: do we need this to be public in the end?

Reply:
Not really...